src/shared/app.h

```diff
@@ -50,6 +50,10 @@ class App : public QObject {
     // The user needs to accept the telemetry policy.
     StateTelemetryPolicy,
 
+    // The user was successfully authenticated and moves into the onboarding
+    // flow
+    StateOnboarding,
```

Comment:
Does this need a corresponding state for the frontend? For example in ControllerImage.qml we account for other states and what the UI should do. This might be different so just checking

Reply:
Hmm, not sure what you mean by "corresponding state for the frontend". We use this app state to tell the navigator to load `ScreenOnboarding.qml` once we are in `StateOnboarding`, but we aren't changing any UI elsewhere based on being in the onboarding state.